example/package.json

```diff
@@ -71,7 +71,7 @@
     "react-native-background-timer": "^2.4.1",
     "react-native-bootsplash": "^3.2.3",
     "react-native-calendar-picker": "^7.1.2",
-    "react-native-calendar-strip": "^2.2.5",
+		"react-native-calendar-strip-fixed": "^2.2.5-updated",
```

Comment:
Please add some docs or explanation about this change

Reply:
solved issue of library https://github.com/BugiDev/react-native-calendar-strip/issues/306